docs/installation/production/prerequisites.md

```diff
@@ -1,104 +1,109 @@
-    usually for consumer hardware this can not be automated
-    (it requires something like [IPMI](https://en.wikipedia.org/wiki/Intelligent_Platform_Management_Interface) to automate).
-
-Common settings:
-
-- Enable Wake-on-LAN (WoL) and network boot
-- Use UEFI mode and disable CSM (legacy) mode
-- Disable secure boot
-
-Boot order options (select one, each has their pros and cons):
-
-1. Only boot from the network if no operating system found: works on most hardware but you need to manually wipe your hard drive or delete the existing boot record for the current OS
-2. Prefer booting from the network if turned on via WoL: more convenience but your BIOS must support it, and you must test it throughly to ensure you don't accidentally wipe your servers
-
-!!! example
-
-    Below is my BIOS setup for reference. Your motherboard may have a different name for the options, so you'll need to adapt it to your hardware.
-
-    ```yaml
-    Devices:
-      NetworkSetup:
-        PXEIPv4: true
-        PXEIPv6: false
-    Advanced:
-      CPUSetup:
-        VT-d: true
-    Power:
-      AutomaticPowerOn:
-        WoL: Automatic  # Use network boot if Wake-on-LAN
-    Security:
-      SecureBoot: false
-    Startup:
-      CSM: false
-    ```
-
-## Gather information
-
-- [ ] MAC address for each machine
-- [ ] OS disk name (for example `/dev/sda`)
-- [ ] Network interface name (for example `eth0`)
-- [ ] Choose a static IP address for each machine (just the desired address, we don't set anything up yet)
+# Prerequisites
+
+## Fork this repository
+
+Because [this project](https://github.com/khuedoan/homelab) applies GitOps practices,
+it's the source of truth for _my_ homelab, so you'll need to fork it to make it yours:
+
+[:fontawesome-solid-code-fork: Fork khuedoan/homelab](https://github.com/khuedoan/homelab/fork){ .md-button }
+
+By using this project you agree to [the license](/license).
+
+
+!!! summary "License TL;DR"
+
+     - This project is free to use for any purpose, but it comes with no warranty
+     - You must use the same [GPLv3 license](https://www.gnu.org/licenses/gpl-3.0.en.html)  in `LICENSE.md`
+     - You must keep the copy right notice and/or include an acknowledgement
+     - Your project must remain open-source
+
+## Hardware requirements
+
+### Initial controller
+
+!!! info
+
+    The initial controller is the machine used to bootstrap the cluster, we only need it once, you can use your laptop or desktop
+
+- A Linux machine that can run Docker (because the `host` networking driver used for PXE boot [only supports Linux](https://docs.docker.com/network/host/), you can use a Linux virtual machine with bridged networking if you're on macOS or Windows).
+- Many linux distributions, such as Fedora, deploy with SELinux enabled and enforced. Configure this appropriately on your own or simply set to permissive (at least it'll be logging):
+```
+sed -i s/^SELINUX=.*$/SELINUX=permissive/ /etc/selinux/config
+setenforce 0
+```

```

Comment:
```suggestion
- Some Linux distributions, such as Fedora, deploy with SELinux enabled and enforced. Configure this appropriately on your own or simply [set it to permissive](https://docs.fedoraproject.org/en-US/quick-docs/changing-selinux-states-and-modes/#selinux-changing-to-permissive-mode).
```

We should link this to the official document instead, which contains more information and context.